spec.html

```diff
@@ -138,9 +154,9 @@ <h1>Static Semantics: UnicodeMatchProperty ( _p_ )</h1>
   <emu-import href="table-nonbinary-unicode-properties.html"></emu-import>
   <p>Additionally, implementations must support the following binary Unicode properties and their property aliases:</p>
   <emu-import href="table-binary-unicode-properties.html"></emu-import>
-  <p>The abstract operation UnicodeMatchProperty takes a string parameter _p_ and performs the following steps:</p>
+  <p>The abstract operation UnicodeMatchProperty takes a parameter _p_ that is a List of |SourceCharacter| values and performs the following steps:</p>
   <emu-alg>
-    1. Assert: _p_ is a known Unicode property name or property alias as listed in the “Property name and aliases” column of <emu-xref href="#table-nonbinary-unicode-properties"></emu-xref> or <emu-xref href="#table-binary-unicode-properties"></emu-xref>.
+    1. Assert: _p_ is a Unicode property name or property alias listed in the “Property name and aliases” column of <emu-xref href="#table-nonbinary-unicode-properties"></emu-xref> or <emu-xref href="#table-binary-unicode-properties"></emu-xref>.
```

Comment:
like static semantic rules above.  But since this is just an assert it probably is ok to say it in this sloppier (and more concise) manner